packages/kamino-sdk/src/Kamino.ts

```diff
@@ -1230,10 +1238,21 @@ export class Kamino {
       decimalsA,
       decimalsB
     );
+    let lowerResetPrice: Decimal | null = null;
+    let upperResetPrice: Decimal | null = null;
+    if (rebalanceKind.kind === PricePercentageWithReset.kind) {
+      const state = deserializePricePercentageWithResetRebalanceFromOnchainParams(poolPrice, strategy.rebalanceRaw);
+      lowerResetPrice = new Decimal(state.find((param) => param.label == 'resetPriceLower')?.value.toString()!);
```

Comment:
If there is no resetPriceLower param in state, this will fail (e.g. `new Decimal(undefined)`), can you maybe add a check for this and throw a friendlier error if it's not there?

Usually these errors are hard to read from logs as it usually throws something like `undefined not a number`

Reply:
if the strategy is `PricePercentageWithReset` or `Expander` it should always have a reset lower and upper and we should throw if not, but yeah, agree that the err should be nicer